package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "roosterjs",
-  "version": "6.16.3",
+  "version": "6.16.4",
```

Comment:
Would you please upgrate to 6.17.0 since I also added some new APIs.

Reply:
sure